test/fixtures/vcr_cassettes/github_create_file.json

```diff
@@ -126,9 +157,9 @@
         "X-Content-Type-Options": "nosniff",
         "X-Frame-Options": "deny",
         "X-XSS-Protection": "1; mode=block",
-        "X-Runtime-rack": "0.628030",
-        "X-Served-By": "b535085e7f4d6e3423e016e684de0829",
-        "X-GitHub-Request-Id": "EEB6:5560:14AD7B:17F6C0:5941B740"
+        "X-Runtime-rack": "0.673903",
+        "X-Served-By": "2c18a09f3ac5e4dd1e004af7c5a94769",
+        "X-GitHub-Request-Id": "D998:7CB5:E5FF45:11508EC:595E353D"
```

Comment:
We can but is that really the issue here?

There are a lot more parameter changes and this happened because I had to use a new (shared) token to recreate the cassettes instead of using an individual one. With the new token, you probably won't need to recreate them again.